Sources/View/UICollectionView+Reusable.swift

```diff
@@ -18,8 +18,8 @@ public extension UICollectionView {
 
    - seealso: `registerNib(_:,forCellWithReuseIdentifier:)`
    */
-  final func registerReusableCell<T: UICollectionViewCell where T: NibReusable>(cellType: T.Type) {
-    self.registerNib(cellType.nib, forCellWithReuseIdentifier: cellType.reuseIdentifier)
+  final func registerReusableCell<T: UICollectionViewCell>(_ cellType: T.Type) where T: NibReusable {
```

Comment:
Wondering if, to comply to the Swift 3.0 API design guidelines, we shouldn't declare that as `register<T: UICollectionViewCell>(cellType: T.Type) where T: NibReusable` instead (so that call site looks like `collectionView.register(cellType: MyXIBCell.self)`?

Of course same comment and reflection applies to all the methods for `UICollectionView` and `UITableView`.